bittensor/utils/networking.py

```diff
@@ -151,29 +151,6 @@ class UPNPCException(Exception):
     """ Raised when trying to perform a port mapping on your router. """
 
 
-def upnpc_delete_port_map(external_port: int):
```

Comment:
Is this also not being used, im suprised.

Reply:
Nope.